nesta/production/orms/cordis_h2020_orm.py

```diff
@@ -0,0 +1,126 @@
+'''
+CORDIS H2020
+============
+'''
+
+from sqlalchemy import Table, Column, ForeignKey
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+from sqlalchemy.dialects.mysql import VARCHAR, TEXT
+from sqlalchemy.types import DATETIME, INTEGER, FLOAT, BOOLEAN
+
+Base = declarative_base()
+
+
+class ProjectProgrammes(Base):
+    __tablename__ = 'cordisH2020_project_programmes'
+    project_rcn = Column(INTEGER, 
+                         ForeignKey('cordisH2020_projects.rcn'), 
+                         primary_key=True)
+    programme_code = Column(VARCHAR(19, collation='utf8_bin'),
+                            ForeignKey('cordisH2020_programmes.id'), 
+                            primary_key=True)
+
+class Programme(Base):
+    __tablename__ = 'cordisH2020_programmes'
+    id = Column(VARCHAR(19, collation='utf8_bin'), primary_key=True)
+    framework_programme = Column(VARCHAR(5, collation='utf8_bin'))
+
+class Organizations(Base):
+    __tablename__ = 'cordisH2020_organizations'
+    project_rcn = Column(INTEGER, index=True)
+    #project_id = Column(INTEGER)
+    #project_acronym = Column(TEXT(collation='utf8_bin'))
+    role = Column(VARCHAR(15, collation='utf8_bin'))
+    id = Column(INTEGER, primary_key=True, autoincrement=False)
+    name = Column(TEXT(collation='utf8_bin'))
+    short_name = Column(TEXT(collation='utf8_bin'))
+    activity_type = Column(VARCHAR(3, collation='utf8_bin'))
+    end_of_participation = Column(BOOLEAN)
+    ec_contribution = Column(FLOAT)
+    country = Column(VARCHAR(2))
+    street = Column(TEXT(collation='utf8_bin'))
+    city = Column(TEXT(collation='utf8_bin'))
+    post_code = Column(VARCHAR(20, collation='utf8_bin'))
+    organization_url = Column(TEXT(collation='utf8_bin'))
+    vat_number = Column(VARCHAR(20, collation='utf8_bin'))
+    contact_form = Column(TEXT(collation='utf8_bin'))
+    contact_type = Column(VARCHAR(14, collation='utf8_bin'))
+    contact_title = Column(VARCHAR(5, collation='utf8_bin'))
+    contact_first_names = Column(VARCHAR(12, collation='utf8_bin'))
+    contact_last_names = Column(VARCHAR(14, collation='utf8_bin'))
+    contact_telephone_number = Column(VARCHAR(17, collation='utf8_bin'))
+    contact_fax_number = Column(VARCHAR(17, collation='utf8_bin'))
```

Comment:
would it make more sense just to set the whole table collation? ` __table_args__ = {'mysql_collate': 'utf8_bin'}
` 

It will mean all the tables have to be collated otherwise the foreign key constraints can't be set up. 

I appreciate that it may take up more space, so maybe not worth it, but it would be neater.